package.json

```diff
@@ -51,10 +51,12 @@
     "i18next-browser-languagedetector": "2.1.0",
     "i18next-xhr-backend": "1.5.0",
     "immutable": "3.8.2",
+    "lodash.throttle": "4.1.1",
```

Comment:
I'd rather import lodash with all the utilities/features it includes.
Its very powerful tool to have and Im positive we will use it.